pkgs/development/beam-modules/build-mix.nix

```diff
@@ -23,7 +24,7 @@ let
 
   pkg = self: stdenv.mkDerivation (attrs // {
     name = "${name}-${version}";
-    inherit version src buildInputs;
+    inherit version src;
```

Comment:
just putting it here so we don't forget.
Those changes were merged in a separate PR so you'll probably need to rebase

Reply:
done